### DISCUSSION TITLE
Event data for Bitcoin's "accountsChanged" is always an empty array

### DISCUSSION THREAD

**Discussion by nbran** [phantom/docs#220] [Help & Support] [answered]
### Summary

When changing between Bitcoin-enabled accounts, for an already _"Connected App"_, the `accountsChanged` event always returns an empty array.

Additionally, it would be helpful to understand how many events are emitted for any given event instance as in this example it happens 4x per event.

### Example

https://stackblitz.com/edit/vitejs-vite-ghwkeu?file=src%2FApp.tsx&terminal=dev

### Steps to Reproduce

1. Open console to monitor log outputs
2. Add a multi-chain account **A**
3. Connect the dApp to account **A** using the "Connect" buttons (the console will log the result of the connection attempt)
4. Add a multi-chain account **B**
5. Connect the dApp to account **B** using the "Connect" buttons (the console will log the result of the connection attempt)
6. After a successfully connecting the dApp to both accounts, switch accounts
7. Notice that the console events for Solana and Ethereum will contain data, but the event for Bitcoin will be an empty array - Despite the accounts supporting Bitcoin and the dApp already connected/authorized.

### Phantom Version

24.5.0

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by nbran** [comment]
No change as of v24.6.0

**Reply by adamdelphantom** [reply]
@nbran Thanks for reporting! We are looking into this issue.

**Reply by adamdelphantom** [reply, marked as answer]
@nbran This should be addressed in the next version. Thanks again for reporting!
